scss/forms/_questions.scss

```diff
@@ -0,0 +1,29 @@
+@import "_colours";
+@import "_typography.scss";
+
+.question {
+  margin: 15px 0 0 0;
+  clear: both;
+}
+
+.first-question {
+  margin-top: 0;
+}
```

Comment:
Should we use `@extend` here? I.e. in the markup, which is preferable:
1. `class="question first-question"`
2. `class="first-question"`

If 2., then I think for consistency `question-first` is more in keeping with `banner-warning`, `banner-success` etc.